Symfony/CS/Tokenizer/Tokens.php

```diff
@@ -975,7 +987,7 @@ public function isShortArray($index)
 
         $prevToken = $this[$this->getPrevMeaningfulToken($index)];
 
-        if (!$prevToken->equalsAny(array(']', '}', ')', array(T_STRING), array(T_VARIABLE), array(T_ARRAY_CAST)))) {
```

Comment:
Look at the first diff comment (outdated): https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1060#discussion-diff-26448154